### DISCUSSION TITLE
"SlowDown: Resource requested is unreadable, please reduce your rate" error

### DISCUSSION THREAD

**Discussion by mshanmu** [minio/minio#14717] [Q&A]
MinIO Version: RELEASE.2022-01-08T03-11-54Z

When I setup Loki to ingest logs, it keeps failing with msg="failed to flush user" err="SlowDown: Resource requested is unreadable, please reduce your rate\n\tstatus code: 503" for PutObject from Loki.

I understand this is due to too many requests from Loki to that particular bucket, since PutObject to other buckets went fine at the same time. So, my questions,

1. Is there a configuration parameter to increase the number of requests per bucket?
2. Is there a backoff mechanism for it to succeed? 
3. Is this error getting propagated from the Disk IO or Is this implemented using throttling on a per bucket basis?

**Comment by harshavardhana** [comment]
Please upgrade

**Reply by mshanmu** [reply]
Thanks, @harshavardhana  !! Will do the upgrade. But, can you please explain how the decision is made to return the "slowdown" error message?

**Reply by harshavardhana** [reply]
> Thanks, @harshavardhana !! Will do the upgrade. But, can you please explain how the decision is made to return the "slowdown" error message?

That means you lost quorum, overloaded the server.

**Reply by mshanmu** [reply]
That would mean all the buckets would be affected, right? I saw only one bucket being affected.

**Reply by harshavardhana** [reply]
it means that whichever erasure set it was doing I/O was affected, not necessarily it would be mean it would affect everyone. Quorum losses when the network is involved can be temporary - it simply points to your network is overloaded at some point.

**Reply by mshanmu** [reply]
Mmm.. it is a little odd, since the same erasure set is being used by other buckets as well and they are not affected.

**Reply by harshavardhana** [reply]
Then its the object that is getting overwritten has an issue.

**Reply by dunghn87** [reply]
Is there a way to restore the object when the hardware fails?
Here my cluster http://minio{1...4}/data/disk{1...4} http://minio{5...8}/data/disk{1...4}

minio1: not found
minio2: /data/disk{1..4}/butket/objectname/xl.meta
minio3: /data/disk{1..4}/butket/objectname/xl.meta
minio4: not found

minio5: not found
minio6: not found
minio7: not found
minio8: not found

I Lost Quorum 
MinIO VERSION 2022-02-05T04:40:59Z

**Reply by Ri1a** [reply]
Still getting this error sometimes using bitnami/minio:2022.10.24-debian-11-r0 (25.10.22) with Minio distributed mode.

**Reply by harshavardhana** [reply]
We do not support bitnami containers please use our officially published containers.
